### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by Marmontt**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
+import TodoMenu from "./components/TodoMenu";
+import TodoList from "./components/TodoList";
 import './App.css';
 
+document.body.style.background = '#ebf3f6';
```
Comment (ekalkutin) [inline src/App.js]: should be moved to CSS/SCSS file

File: src/components/TodoList/TodoCreator/index.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.todoCreator-line {
+    position: relative;
+    flex-grow: 1;
+    height: 1px;
+}
+
+.todoCreator-text {
+    position: relative;
```
Comment (ekalkutin) [inline src/components/TodoList/TodoCreator/index.css]: why this is needed?

File: src/components/TodoList/TodoCreator/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import './index.css';
+import * as todoActions from "../../../state/todos/action";
+import {connect} from 'react-redux';
+
+const TodoCreator = ({color, text, fulfilled, todoIndex, setFulfilledState}) => {
+
+    return (
+        <div style={{background: color}} className={'todoCreator-container'}>
+            {fulfilled === true ?
+                <hr className={'todoCreator-line'} color={'#000'}/>
```
Comment (ekalkutin) [inline src/components/TodoList/TodoCreator/index.js:14]: use  `text-decoration: line-through`

File: src/components/TodoList/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import TodoCreator from "./TodoCreator";
+import './index.css';
+import {getAllTodos} from "../../state/selectors";
+import {useSelector} from 'react-redux';
```
Comment (ekalkutin) [inline src/components/TodoList/index.js]: imports from node_modules move to top
`import React, {useState} from 'react';`
`import {useSelector} from 'react-redux';`

` ....rest `

File: src/components/TodoList/index.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                <div className={'todoList-fulfilled-container'}>
+                    <span className={'todoList-fulfilled'}>Show fulfilled todos:</span>
+                    <label className={'switch'}>
+                        <input type={'checkbox'} onChange={() => setToggleState(!toggleState)} checked={toggleState}/>
+                        <span className={'slider round'}/>
+                    </label>
+                </div>
+            </div>
+            {
+                toggleState === false ?
```
Comment (ekalkutin) [inline src/components/TodoList/index.js]: create selector for filtering todos

**Review by ekalkutin — CHANGES_REQUESTED** [review]

File: src/components/TodoList/index.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+                    todosList
+                        .map(({text, color, fulfilled}, index) =>
+                            <TodoCreator todoIndex={index} key={index} color={color} text={text}
+                                         fulfilled={fulfilled}/>)
+                        .filter(todoCreator => todoCreator.props.fulfilled === false) :
```
Comment (ekalkutin) [inline src/components/TodoList/index.js]: use selector

**Review by ekalkutin — CHANGES_REQUESTED** [review]

File: src/state/todos/reducer.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+                    ...state.todos,
+                    {
+                        text: action.payload.text,
+                        color: action.payload.color,
+                        fulfilled: action.payload.fulfilled
+                    }
+                ]
+            };
+        case types.REMOVE_TODO_SUCCESS:
+            todos = state.todos.slice();
```
Comment (ekalkutin) [inline src/state/todos/reducer.js]: this is not needed

**Review by ekalkutin — CHANGES_REQUESTED** [review]

File: src/components/TodoList/TodoCreator/index.js (hunk 3 above)
Comment (Marmontt) [reply src/components/TodoList/TodoCreator/index.js:14]: i guess we should keep my realization without changes!

File: src/App.js [hunk 8]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+
 import TodoMenu from "./components/TodoMenu";
 import TodoList from "./components/TodoList";
+
 import './App.css';
 
-document.body.style.background = '#ebf3f6';
+// document.body.style.background = '#ebf3f6';
```
Comment (ekalkutin) [inline src/App.js:8]: remove comment

**Pull request closed without merging**